src/custom/pages/Swap/index.tsx

```diff
@@ -24,12 +29,57 @@ const BottomGrouping = styled(BottomGroupingUni)`
   }
 `
 
+const SwapModWrapper = styled(SwapMod)`
+  ${props => props.className} {
+    // For now to target <SwapHeader /> without copying files...
+    > div:first-child {
```

Comment:
Here I just pick the first `div` to target <SwapHeader /> to save some trouble copying/modding extra files.